### ISSUE TITLE
ALMNH - Collection Portal Requests

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/new-collections#229]
Institution contact requests collection portal creation. Please use the form at [Reports/CollectionCreationRequest](http://arctos.database.museum/new_collection.cfm)

**Comment by Jegelewicz** [comment]
Lam,

ALMNH already has one portal in Arctos and they have just been approved for four more. Along with these four, can you also create an ALMNH institutional portal for them?

Thanks!

**Comment by Jegelewicz** [comment]
@lkvoong many apologies! I just discovered that ALMNH needs three more collection portals. I have added them in manage collections and approved to create them. I can't do anything with their previously created collections, so all of that is still in there. The three new ones are 

ALMNH:Mamm
ALMNH:Bird
ALMNH:Herp

Let me know if you need anything else to create these and please add them to their institutional portal.

Thank you!!!

**Comment by Jegelewicz** [comment]
@lkvoong can we get the collections set up for ALMNH?

ALMNH:Mamm
ALMNH:Bird
ALMNH:Herp

**Comment by Jegelewicz** [comment]
@dustymc @lkvoong can you guys get these portals created? They say "complete" in Arctos, but they aren't showing up on the portals page or in search.

Thanks!

**Comment by lkvoong** [comment]
@Jegelewicz 
Hi Teresa,

Sorry, I was confused by the previously created collections in the list of collections that needed to b created, and missed that there were 3 new collections that needed to be created for the institution. 

I've just created them:
- http://arctos.database.museum/almnh_bird
- http://arctos.database.museum/almnh_herp
- http://arctos.database.museum/almnh_mamm

For these three collections, I've granted access to you and Mary Beth Prondzinski (login: mbprondzinski).

I've added them to the institution portal, but they aren't showing up yet, due to caching.  As usual, it may take up to 24 hours:
- http://arctos.database.museum/almnh_all

Let me know if there's anything I missed.

Thanks!
